.github/workflows/check-release-version.yaml

```diff
@@ -0,0 +1,141 @@
+name: Check version number
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: Version number to check
+        type: string
+        required: true
+
+    outputs:
+      version:
+        description: The validated version number
+        value: ${{ jobs.check-release-number.outputs.version }}
+      
+    secrets:
+      SLACK_WEBHOOK:
+        description: Slack Notifier Incoming Webhook
+        required: true
+
+jobs:
+  # Check release number
+  check-release-number:
+    runs-on: ubuntu-latest
```

Comment:
does this needs to run on ubutu or macosx ?

Reply:
It doesn't matter, actually... For the purpose of this job, both would work...